managed/src/main/java/com/yugabyte/yw/commissioner/tasks/UniverseTaskBase.java

```diff
@@ -314,8 +315,16 @@ public SubTaskGroup createDestroyServerTasks(Collection<NodeDetails> nodes,
       // Check if the private ip for the node is set. If not, that means we don't have
       // a clean state to delete the node. Log it and skip the node.
       if (node.cloudInfo.private_ip == null) {
-        LOG.warn(String.format("Node %s doesn't have a private IP. Skipping node delete.",
-                               node.nodeName));
+        LOG.warn(String.format("Node %s doesn't have a private IP. Skipping node delete.", node.nodeName));
```

Comment:
1. We have line length limited to 100 characters. Please update this line and check link 320.
2. Not sure that this text fully reflects the updated behaviour.

Reply:
> Not sure that this text fully reflects the updated behaviour.

We are skipping node delete for other providers but if it is onprem then we are clearing the existing table entry.
Please suggest a better warning.